### PR TITLE
fix(button-modal): add header slot

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [x] Tested with console

### Description
Add `header` slot to PButtonModal for flexible customizing.

### CODE REVIEW

**Review by JeoungSulMo — APPROVED** [review]
L G T M ! ! !

File: src/feedbacks/modals/button-modal/PButtonModal.vue [hunk 1]
```diff
@@ -13,14 +13,16 @@
...
-                            <div v-if="!hideHeader" class="modal-header">
-                                <span class="alert-icon">
-                                    <p-lottie name="lottie_error" auto :size="1.5"
-                                              :class="[`modal-${themeColor}`]" class="header-lottie"
-                                    />
-                                </span>
-                                <span>{{ headerTitle }}</span>
-                            </div>
+                            <slot name="header">
+                                <div v-if="!hideHeader" class="modal-header">
```
Comment (WANZARGEN) [inline src/feedbacks/modals/button-modal/PButtonModal.vue:17]:
```suggestion
                            <slot  v-if="!hideHeader" name="header">
                                <div class="modal-header">
```

I think it must be better!

**Review by WANZARGEN — APPROVED** [review]
I left a comment~!

**Pull request merged**